docs/index.md

```diff
@@ -133,3 +133,4 @@ We love to hear from our users and help them be successful with ddev. Support op
 * [ddev-contrib](https://github.com/drud/ddev-contrib) repo provides a number of vetted user-contributed recipes for extending and using ddev. Your contributions are welcome.
 * [awesome-ddev](https://github.com/drud/awesome-ddev) repo has loads of external resources, blog posts, recipes, screencasts, and the like. Your contributions are welcome.
 * [Twitter with tag #ddev](https://twitter.com/search?q=%23ddev&src=typd&f=live) will get to us, but it's not as good for interactive support, but we'll answer anywhere.
+https://my.typo2.org/index.php?id=35
```

Comment:
I don't understand what you're adding here - I assume it's TYPO3-related, but please explain what you want to accomplish.  TYPO3 slack is listed above...